inc/buddyboss.php

```diff
@@ -329,3 +329,43 @@ function nightingale_theme_following_count( $user_id = false ) {
 	}
 }
 
+
+if ( ! function_exists( 'buddyboss_pagination' ) ) {
+
+	/**
+	 * Custom Pagination
+	 */
+	function buddyboss_pagination() {
+		global $paged, $wp_query;
+
+		$max_page = 0;
+
+		if ( ! $max_page ) {
+			$max_page = $wp_query->max_num_pages;
+		}
+
+		if ( ! $paged ) {
+			$paged = 1;
```

Comment:
should it be a ```$page``` ? rather than ```$paged``` which suggests a flag variable with a boolean value ... Page describes exactly which page we are talking about

Reply:
That would be infinitely more logical. Unfortunately $paged is a variable carried in from the plugin